From 0786e4d401640ed9d75976342c87fcd2315b6949 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 22 Sep 2024 22:22:23 +0800
Subject: [PATCH] feat(.desktop): add Keywords= entries

This should make it easier to search for the game in non-English environments.
---
 pkg/appstream/net.wz2100.warzone2100.desktop.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pkg/appstream/net.wz2100.warzone2100.desktop.in b/pkg/appstream/net.wz2100.warzone2100.desktop.in
index 46ba23d18..812b0e2fa 100644
--- a/pkg/appstream/net.wz2100.warzone2100.desktop.in
+++ b/pkg/appstream/net.wz2100.warzone2100.desktop.in
@@ -52,3 +52,5 @@ Comment[pt]=Estratégia em tempo real em ambiente pós-nuclear.
 Comment[zh_CN]=核心实时策略
 Name[zh_CN]=战地2100@WZ_NAME_SUFFIX@
 GenericName[zh_CN]=战略游戏
+Keywords=warzone;strategy;rts;real-time;
+Keywords[zh_CN]=战地;战略;实时战略游戏;warzone;strategy;rts;real-time;
-- 
2.46.1

